app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -467,7 +469,7 @@ export default Component.extend(TextareaTextManipulation, {
         /* offline */ true
       );
 
-      resolveAllShortUrls(ajax, this.siteSettings, cookedElement);
+      resolveCachedShortUrls(this.siteSettings, cookedElement);
```

Comment:
Before morphing, we only use the cached results so it's duper fast. For those who aren't already loaded, they will be in the "previewUpdated" later.